core/tests/performance_tests/base.py

```diff
@@ -36,9 +40,22 @@ def _record_page_metrics_from_cached_session(self, page_url):
             self.data_fetcher.get_page_metrics_from_cached_session(page_url))
 
     def _record_page_timings_for_url(self, page_url):
-        self.page_metrics = (
-            self.data_fetcher.get_page_timings_for_url(page_url))
+        page_session_metrics = []
+
+        for _ in range(self.SESSION_COUNT):
+            page_session_metrics.append(
+                self.data_fetcher.get_page_timings_for_url(page_url))
+
+        self.page_metrics = perf_domain.MultiplePageSessionMetrics(
+            page_session_metrics)
 
     def _record_page_timings_from_cached_session(self, page_url):
```

Comment:
I think you misunderstood me. I was fine with you keeping ``session_count`` here, but I was thinking something like:

```python
def _record_page_timings_from_cached_session(self, page_url, session_count=DEFAULT_SESSION_COUNT)
```

When you call this method, you don't need to specify ``session_count``. My suggestion was for you to not specify ``session_count`` when calling ``_record_page_timings_from_cached_session``--I wasn't suggesting you remove the ``session_count`` argument altogether.